test/cpp/jit/test_gpu.cpp

```diff
@@ -15292,6 +15292,30 @@ TEST(NVFuserTest, TestSegmentIslands_CUDA) {
   fusion_executor_cache.runFusionWithInputs({t0, t1});
 }
 
+TEST(NVFuserTest, FusionSimpleBcastBack) {
+  auto fusion_ptr = std::make_unique<Fusion>();
+  auto fusion = fusion_ptr.get();
+  FusionGuard fg(fusion);
+
+  auto tv0 = makeSymbolicTensor(2);
+  fusion->addInput(tv0);
+  auto tv1 = sum(tv0, {1});
+  auto tv2 = broadcast(tv1, {false, true});
+  auto tv3 = add(tv2, tv0);
+  fusion->addOutput(tv3);
+
+  FusionExecutorCache executor_cache(std::move(fusion_ptr));
```

Comment:
We have a lot of correctness tests that will hit this case. All our normalization schedules should. I wouldn't worry about adding a test for correctness. I assume you may be just doing this to check it's working, but wanted to mention you can remove before merging.

Reply:
Yes it was a quick check to see if it works. I have removed this test. Thanks!